sahi/predict.py

```diff
@@ -315,6 +317,10 @@ def predict(
     postprocess_match_threshold: float = 0.5,
     postprocess_class_agnostic: bool = False,
     export_visual: bool = False,
+    input_video: bool = False,
+    view_image: bool = False,
+    fast_forwarding: bool = False,
+    save_dir: str = None,
```

Comment:
this parameter is not necessary, we can remove it, save_dir is generated as `{project}/{name}` in line 435

Reply:
That's true. I removed it.